src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -647,11 +650,11 @@ public static String sanitizeFolderNames(String zipFileName) {
 	 */
 	public static String modifyFileNameOnType(String type, String fileName) {
 		String fileExtension = FilenameUtils.getExtension(fileName);
-		if ((fileExtension == null 
-			|| fileExtension.isEmpty() 
+		if ((fileExtension == null
```

Comment:
Please verify that this big complicated test formats clearly if someone does automatic re-indentation on the code.